modules/jt-comparing-jenkins-tekton-execution-models.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assembly:
+//
+// jenkins-tekton/migrating-from-jenkins-to-tekton.adoc
+
+[id="jt-comparing-jenkins-tekton-execution-models_{context}"]
+= Comparing Jenkins and Tekton execution models
+
+Jenkins and Tekton offer similar functions, but are architecturally different and execute things differently. This section outlines a brief comparison of the two execution models.
+
+.Comparison of execution models in Jenkins and Tekton
+[cols="1,1",options="header"]
+|===
+|Jenkins|Tekton
+|Jenkins has a control node and executes pipelines and steps centrally, or orchestrates jobs running in other nodes|Tekton is serverless and distributed, and there is no central dependency for execution.
```

Comment:
Jenkins has a control node and executes pipelines and steps centrally, or orchestrates jobs running in other nodes --> Jenkins has a control node. Jenkins executes pipelines and steps centrally, or orchestrates jobs running in other nodes

Reply:
other nodes --> other nodes.

Reply:
Observation and suggestion: It is nice to have the comparison statements written in a similar pattern, so the reader feels connected when understanding the comparison. Doing so will improve the reading flow and intelligibility.

Reply:
It's very difficult to do so here without tampering the technical meaning, because the different aspects of the models are not exactly comparable. They have vague similarities - that's it.